content/blog/2024/climaterisk-scipy-tutorial/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Determining Climate Risks with NASA Earthdata Cloud at Scipy 2024 
+date: "2024-07-08"
+banner:
+  image: "featured.png"
+authors: ["Jenny Wong"]
+tags: [geoscience, education]
+categories: [impact]
+featured: false
+draft: false
+---
+
+*Determining Climate Risks with NASA Earthdata Cloud* is [ScienceCore](https://www.nasa.gov/centers-and-facilities/marshall/nasa-boosts-open-science-through-innovative-training/) curriculum module that comprises part of NASA's [Open Science](https://science.nasa.gov/open-science-overview) and [Transform to Open Science (TOPS)](https://nasa.github.io/Transform-to-Open-Science/) initiatives. The aim of this module is to deliver a hands-on experience with "data-proximate computing" in the cloud with [NASA Earthdata](https://www.earthdata.nasa.gov/) products with content co-developed with MetaDocencia.
+
+This module was delivered as a [SciPy tutorial](https://cfp.scipy.org/2024/talk/3DVH7S/) at this year's conference. 2i2c have been working closely with the organizers to provide the hub infrastructure for the tutorial, including enabling a shared password for easy authentication (see our [Openscapes post](/blog/2024/openscapes-sbg-workshop/index) for more about this feature) and operating a [small binder service](/blog/2024/nasa-ephemeral-hubs/index) for participants to view content after the event (you can take a look at the tutorial on the [NASA ephemeral hub](https://binder.opensci.2i2c.cloud/v2/gh/ScienceCore/scipy-2024-climaterisk/HEAD?labpath=00_Introduction_Setup%2F01_Initial_Setup.ipynb)!).
+
+The event was well-attended, with 40 learners taking part. Special thanks go to the organizers Dhavide Aruliah, Karthik Venkataramani and Patricia A. Loto for leading the tutorial.
```

Comment:
```suggestion
The event was well-attended, with 40 learners taking part. Special thanks go to the organizers [Dhavide Aruliah](https://github.com/dhavide), [Karthik Venkataramani](https://www.linkedin.com/in/karthikvenkataramani/) and [Patricia A. Loto](https://github.com/patriloto) for leading the tutorial.
```

Reply:
I try to link to individuals wherever possible in case this helps signal-boost their profile